src/test/regress/sql/pg15.sql

```diff
@@ -269,6 +269,25 @@ SELECT * FROM numeric_scale_gt_precision ORDER BY 1;
 -- verify we can route queries to the right shards
 SELECT * FROM numeric_scale_gt_precision WHERE numeric_column=0.027;
 
+-- test repartition joins on tables distributed on numeric types with negative scale
+CREATE TABLE numeric_repartition_first(id int, numeric_column numeric(2,3));
```

Comment:
these are not negative scale numerics?

Reply:
Agreed. I updated the scales now. While doing that I realized that there is another bug associated with numeric columns. See https://github.com/citusdata/citus/pull/6358/files#r978063398